viscy/light/engine.py

```diff
@@ -798,4 +809,4 @@ def on_predict_epoch_end(self) -> None:
             }
         )
 
-        df.to_csv("embeddings2/multi_resnet_predicted_metadata.csv", index=False)
+        df.to_csv("embeddings4/1_multi_resnet_predicted_metadata.csv", index=False)
```

Comment:
This should be configurable. No change to library code should be needed just for another training run.